x-pack/plugins/fleet/server/routes/setup/index.ts

```diff
@@ -40,11 +78,25 @@ export const registerCreateFleetSetupRoute = (router: FleetAuthzRouter) => {
       fleetAuthz: {
         fleet: { setup: true },
       },
+      description: `Initiate agent setup`,
+      options: {
+        tags: ['Elastic Agents'],
+      },
     })
     .addVersion(
       {
         version: API_VERSIONS.public.v1,
-        validate: false,
+        validate: {
+          request: {},
+          response: {
+            200: {
+              body: fleetSetupResponseBody,
```

Comment:
Where seeing inconsistencies between the code and existing openapi spec, I'm going with the code, because that should be more accurate. E.g. openapi for this route refers some non-existent request body (I suppose it was removed at some point): https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/common/openapi/paths/agents%40setup.yaml#L31-L43